lib/test.py

```diff
@@ -188,6 +189,11 @@ def run(self, server):
             if self.is_executed_ok and os.path.isfile(self.result):
                 self.is_equal_result = filecmp.cmp(self.result, self.tmp_result)
             elif self.is_executed_ok:
+                if lib.Options().args.is_verbose:
+                    color_stdout('\n')
+                    with open(self.tmp_result, "r+") as logfile:
```

Comment:
Why 'r+'? Why calling color_stdout() line by line?

Also I think it would be good to add newline in case the file ends w/o endline.

Reply:
> Why calling color_stdout() line by line?

But from the other side of view it allows to process large files.